test/helper.js

```diff
@@ -1,2 +1,21 @@
 import dotenv from 'dotenv';
 dotenv.config();
+
+import { git } from '../src/history/git.js';
+
+let lastCommitId;
+let stashResult;
+
+before(async () => {
+  const { latest: { hash } } = await git.log(['-1']);
+  lastCommitId = hash;
+  stashResult = await git.stash(['-u']);
```

Comment:
```suggestion
  stashResult = await git.stash(['--include-untracked']);
```

Reply:
How can I test changes without committing in such a setup? 🤔

Reply:
It's not possible.